src/Contracts/TagRepository.php

```diff
@@ -87,4 +87,12 @@ public function paginate($tag, $startingAt = 0, $limit = 25);
      * @return void
      */
     public function forget($tag);
+
+    /**
+     * Trim the job IDs from the given tags.
+     *
+     * @param  array  $tags
+     * @return void
+     */
+    public function trim(array $tags);
```

Comment:
We can't add this in a minor release because it'll break implementations who don't have this method defined.

Reply:
That means when `trim` is called it will also break those implementations, since they don't define this method?

Reply:
It'll break as soon as they install this package because the implementation won't adhere to the contract;

Reply:
I mean if we didn't add it to the contract, the `trim` method would fail during runtime because it's probably not defined on the implementation, when it's called on the listener.